### PR TITLE
Update Helm charts as per volume health monitoring feature

### PR DESCRIPTION
**Pull request by geoffrey1330**
This is continuation of the work done as a part of CSI Driver feature: volume health monitoring https://github.com/simplyblock-io/spdk-csi/pull/14

* Updated Helm charts as per volume health monitoring feature

### CODE REVIEW

File: e2e/utils.go [hunk 1]
```diff
@@ -48,61 +48,61 @@ const (
...
-		e2elog.Logf("failed to create config map %s", err)
-	}
-	_, err = framework.RunKubectl(nameSpace, "apply", "-f", secretPath)
-	if err != nil {
-		e2elog.Logf("failed to create secret: %s", err)
-	}
-}
+// func deployConfigs() {
+// 	_, err := framework.RunKubectl(nameSpace, "apply", "-f", configmapPath)
+// 	if err != nil {
```
Comment (boddumanohar) [inline e2e/utils.go:53]: please remove all the commented code.

Comment (geoffrey1330) [reply e2e/utils.go:53]: Alright.

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -33,20 +33,35 @@ jobs:
...
-        sed -i "s/\"uuid\": \".*\"/\"uuid\": \"${{ github.event.inputs.uuid }}\"/" $GITHUB_WORKSPACE/deploy/kubernetes/config-map.yaml
-        sed -i "s/\"ip\": \".*\"/\"ip\": \"${{ github.event.inputs.ip }}\"/" $GITHUB_WORKSPACE/deploy/kubernetes/config-map.yaml
+    # - name: Update ConfigMap
+    #   run: |
+    #     sed -i "s/\"uuid\": \".*\"/\"uuid\": \"${{ github.event.inputs.uuid }}\"/" $GITHUB_WORKSPACE/deploy/kubernetes/config-map.yaml
+    #     sed -i "s/\"ip\": \".*\"/\"ip\": \"${{ github.event.inputs.ip }}\"/" $GITHUB_WORKSPACE/deploy/kubernetes/config-map.yaml
+
+    # - name: Update Secret
+    #   run: |
+    #     sed -i "s/\"secret\": \".*\"/\"secret\": \"${{ github.event.inputs.secret }}\"/" $GITHUB_WORKSPACE/deploy/kubernetes/secret.yaml
```
Comment (boddumanohar) [inline .github/workflows/test.yaml:55]: please also remove this code.

**Review by boddumanohar — APPROVED** [review]
LGTM. 🚀

File: .github/workflows/test.yaml (hunk 2 above)
Comment (geoffrey1330) [reply .github/workflows/test.yaml:55]: Alright

**Pull request merged**